devicon.css

```diff
@@ -232,7 +232,7 @@
 .devicon-twitter-original:before {
   content: "\e91f";
 }
-.devicon-mocha:before {
+.devicon-mocha-plain:before {
```

Comment:
Can you provide a explanation why you suggest this change?

Reply:
So I did this because all css classes follow the same pattern: devicon-{techname}-{version}. I found this when I run the upgrade script because mine was failing at mocha.